### ISSUE TITLE
abnormal somite development

### ISSUE THREAD

**Issue by obophenotype-user** [obophenotype/mammalian-phenotype-ontology#1667]
Consider adding the following terms as synonyms of abnormal somite development [MP:0001688]

SYN: abnormal somitogenesis
SYN: abnormal somite formation
SYN:  abnormal somite segmentation

Also, consider revising the current definition to reflect the following:

SUGGESTED DEF: any anomaly in the formation of mesodermal clusters that are arranged segmentally along the anterior posterior axis of an embryo; somites are transient structures - balls or blocks of paraxial mesoderm cells with an epithelial outer layer and a mesenchymal cell core - that emerge sequentially through a mesenchymal-to-epithelial transition (MET) from the morphologically unpatterned presomitic mesoderm; they give rise to the metameric anatomy of the vertebral column itself, and the associated skin, muscle and tendons

Definition compiled from GO (http://www.informatics.jax.org/searches/GO.cgi?id=GO:0001756) and Nature Reviews Genetics 12, 43-55 (January 2011); doi:10.1038/nrg2902

Thanks

Anna

Reported by: anna_anagnos

Original Ticket: [obo/mammalian-phenotype-requests/1667](https://sourceforge.net/p/obo/mammalian-phenotype-requests/1667)
